doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst

```diff
@@ -9,6 +9,19 @@ Changelog
 
 Notable changes to this controller are documented in this file.
 
+Controller v3357
+****************
+
+The following changes have been introduced in the v3357 of the controller:
+
+Changes
+=======
+
+* Fixed an issue where P0.28 - P0.31 cannot be allocated for controlling FEM.
+* Fixed an issue where the FEM control pins cannot work when TX output power setting is 0 dBm.
+* Be able to read RSSI from CIS ISO channel through HCI_Read_RSSI HCI command.
```

Comment:
```suggestion
* Updated the controller to be able to read RSSI from the CIS ISO channel using the ``HCI_Read_RSSI`` HCI command.
```
This should go to the line 20, that is above the fixes that are currently in lines 20-21. We always list updates before bug fixes.